config/load.go

```diff
@@ -97,7 +97,7 @@ func unmarshallToAppConfig(appConfig *AppConfig) error {
 }
 
 func envConfig() {
-	viper.SetEnvPrefix("pulse")
+	viper.SetEnvPrefix("bhs")
```

Comment:
Now I wonder why we have [declared const](https://github.com/bitcoin-sv/pulse/pull/206/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R19) with env prefix, as we are using a static string here.

Reply:
My bad, I am using const currently